homepage/templates/homepage/post_detail.html

```diff
@@ -1,16 +1,18 @@
-{% extends 'homepage/base.html' %}
+{% extends 'homepage/base.html' %} 
```

Comment:
If/when this starts to bother you, you can switch on the VSCode settings > trim trailing whitespace (automatically gets rid of these). Just turned it on myself :D

Reply:
Thanks! Gonna do it like right now! :D